scripts/__input_gamepad_set_mapping/__input_gamepad_set_mapping.gml

```diff
@@ -202,6 +202,55 @@ function __input_gamepad_set_mapping()
         exit;
     }
     
+    #endregion
+    
+    #region Windows Xbox One Wireless BT (New firmware)
+
+    if ((os_type == os_windows) && (vendor == "5e04")                  //Windows (DirectInput) Microsoft's VID
+    && ((product == "e002") || (product == "fd02"))                    //XbOne Wireless Rev. 1 or 2 PID for BT
+    && (gamepad_button_count(index ) == 17)                            //Probably new firmware based on button
+    && (gamepad_axis_value(index, 1) == gamepad_axis_value(index, 2))  //On new firmware axes 1 and 2 are same
```

Comment:
The idea here is that axes 2 and 5 rest at nonzero on old firmware as well as third party clones that spoof this VID+PID (both are 'good' state). In the unlikely event that a third party device happens to false-positive L211 by presenting 17 buttons, it's further _very unlikely_ the axis pair check will also pass.